### PR TITLE
Add Python Solution for Counting Liars (1228)

### PR DESCRIPTION
**Pull request by jjliewie**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-1228.mdx [hunk 1]
```diff
@@ -83,4 +84,57 @@ int main() {
...
+	g, l, ans = [], [], float('inf')
+	
+	for _ in range(n):
+	
+		a, b = t().split()
+
+		'''
+		The position is stored in g if the cow claims that Bessie is hiding at a location greater than b.
+		The position is stored in l if the cow claims that Bessie is hiding at a location less than b.
+		'''
```
Comment (SansPapyrus683) [inline solutions/usaco-1228.mdx:107]: 1. over 80 chars
2. use double quotes for these multiline string comments

File: solutions/usaco-1228.mdx [hunk 2]
```diff
@@ -83,4 +84,57 @@ int main() {
...
 
 </CPPSection>
+
+<PySection>
+
+```python
+import sys
+t = sys.stdin.readline
+
+def solution():
```
Comment (SansPapyrus683) [inline solutions/usaco-1228.mdx:94]: take out of main plelase

File: solutions/usaco-1228.mdx [hunk 3]
```diff
@@ -83,4 +84,57 @@ int main() {
...
+	
+		a, b = t().split()
+
+		'''
+		The position is stored in g if the cow claims that Bessie is hiding at a location greater than b.
+		The position is stored in l if the cow claims that Bessie is hiding at a location less than b.
+		'''
+
+        	if a == "G": g.append(int(b))
+		else: l.append(int(b))
```
Comment (SansPapyrus683) [inline solutions/usaco-1228.mdx:110]: don't inline this

File: solutions/usaco-1228.mdx [hunk 4]
```diff
@@ -83,4 +84,57 @@ int main() {
...
+        
+	bessie = g + l + [l[0] - 1] + [g[-1] + 1]
```
Comment (SansPapyrus683) [inline solutions/usaco-1228.mdx:112]: what the

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-1228.mdx (hunk 1 above)
Comment (jjliewie) [reply solutions/usaco-1228.mdx:107]: am i not already using double quotes?

File: solutions/usaco-1228.mdx (hunk 2 above)
Comment (jjliewie) [reply solutions/usaco-1228.mdx]: done

File: solutions/usaco-1228.mdx (hunk 4 above)
Comment (jjliewie) [reply solutions/usaco-1228.mdx]: its correct? should i add comment above 

# considers all cases including when Bessie's location is greater than the greatest guess among  the "greater" guesses and when Bessie's location is less than the least guess among "less than" guesses ?

File: solutions/usaco-1228.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-1228.mdx]:
```py
"""
double quotes
"""
```

Comment (jjliewie) [reply solutions/usaco-1228.mdx]: done

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**